### DISCUSSION TITLE
Bing Visual Search button on Windows Search Bar

### DISCUSSION THREAD

**Discussion by shlomova** [valinet/ExplorerPatcher#824] [Ideas]
Bring back the Bing Visual Search button on Windows Search Bar.
[more information]( https://searchengineland.com/microsoft-brings-visual-search-to-windows-search-bar-326522).

**Comment by XuanChen1234** [comment]
@valinet @Amrsatrio

**Reply by valinet** [reply]
@XuanChen1234 You have been blocked on my account for repeatedly spamming my mail inbox with unnecessary tagging.
